### PR TITLE
changed the local host port in aks installation

### LINKED ISSUE #2844: [Docs] The documentation for installing AKS is misleading
**Current State:**
The documentation for installing AKS is misleading and points to port http://localhost:9081

**Desired State:**
The port to the localhost should be corrected to  http://localhost:8080.
Screenshot 2021-05-05 at 10 53 25 AM

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: Anita-ihuman <charlesanita403@gmail.com>

**Description**
changed the localhost port in aks installation to http://localhost:8080
This PR fixes #2844

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
@Anita-ihuman please update the issue number in the description, which this PR addresses.

File: docs/pages/installation/aks.md [hunk 1]
```diff
@@ -75,4 +75,4 @@ to your azure account using *az login*.
  </div></div>
  </pre>
 
-Meshery should now be running in your AKS cluster and the Meshery UI should be locally accessible. Navigate to [http://localhost:9081](http://localhost:9081) to log into Meshery.
+Meshery should now be running in your AKS cluster and the Meshery UI should be locally accessible. Navigate to [http://localhost:8080](http://localhost:8080) to log into Meshery.
```
Comment (leecalcote) [inline docs/pages/installation/aks.md:78]:
```suggestion
Meshery should now be running in your AKS cluster and the Meshery UI should be accessible. Navigate to the `meshery` service endpoint to log into Meshery.

```

Comment (leecalcote) [inline docs/pages/installation/aks.md]:
```suggestion
Meshery should now be running in your AKS cluster and the Meshery UI should be locally accessible. Navigate to the `meshery` service endpoint to log into Meshery.

```

**Review by leecalcote — APPROVED** [review]
Thank you, @Anita-ihuman

**Pull request merged**